Samples/HelloWorld/OrleansHost/OrleansHostWrapper.cs

```diff
@@ -82,88 +80,83 @@ public int Stop()
+                        {
+                            case "deploymentid":
+                                deploymentId = parameters[1];
+                                break;
+                            case "name":
+                                siloName = parameters[1];
+                                break;
+                            default:
+                                Console.WriteLine($"Bad command line arguments supplied: {arg}");
+                                return null;
+                        }
                     }
-                    switch (parameters[0].ToLowerInvariant())
+                    else
                     {
-                        case "deploymentid":
-                            deploymentId = parameters[1];
-                            break;
-                        case "name":
-                            siloName = parameters[1];
-                            break;
-                        default:
-                            Console.WriteLine("Bad command line arguments supplied: " + arg);
-                            return null;
+                        Console.WriteLine($"Bad command line arguments supplied: {arg}");
+                        return null;
                     }
                 }
-                else
-                {
-                    Console.WriteLine("Bad command line arguments supplied: " + arg);
-                    return null;
-                }
+                // Default to machine name
+                siloName = siloName ?? Dns.GetHostName();
+                return new SiloArgs(siloName, deploymentId);
             }
-            // Default to machine name
-            siloName = siloName ?? Dns.GetHostName();
-            return new SiloArgs(siloName, deploymentId); 
-        }
 
-        public void PrintUsage()
-        {
-            string consoleAppName = Assembly.GetExecutingAssembly().GetName().Name;
-            Console.WriteLine(
-                @"USAGE: {0} [name=<siloName>] [deploymentId=<idString>] [/debug]
+            public static void PrintUsage()
+            {
+                string consoleAppName = Assembly.GetExecutingAssembly().GetName().Name;
+                Console.WriteLine(
+                    $@"USAGE: {consoleAppName} [name=<siloName>] [deploymentId=<idString>] [/debug]
                 Where:
                 name=<siloName> - Name of this silo (optional)
                 deploymentId=<idString> - Optionally override the deployment group this host instance should run in 
-                (otherwise will use the one in the configuration", consoleAppName);
-        }
-    }
-
-    class SiloArgs
-    {
-        public SiloArgs(string siloName, string deploymentId)
-        {
-            this.DeploymentId = deploymentId;
-            this.SiloName = siloName;
-        }
+                (otherwise will use the one in the configuration");
+            }
 
-        public SiloArgs()
-        {
```

Comment:
why do we delete this? we require construct SiloArgs with two param ? 
I don't have strong stand on this thought since this class is only used in HostWrapper